Framework/API/src/ScriptBuilder.cpp

```diff
@@ -233,14 +238,24 @@ const std::string ScriptBuilder::buildAlgorithmString(const AlgorithmHistory &al
   }
   // Third case is we never specify the version, so do nothing.
 
+  std::string assignmentStatement("");
+  if (!storeInADS) {
+    properties << "StoreInADS=False, ";
+    const auto it =
+        std::find_if(props.cbegin(), props.cend(), [](const auto &prop) { return prop->name() == "OutputWorkspace"; });
+    if (it != props.cend()) {
+      assignmentStatement = (*it)->value() + " = ";
```

Comment:
The rest of the code in this file seems to not avoid using spaces in the strings containing operators, so maybe its best to do the same here?
```suggestion
      assignmentStatement = (*it)->value() + "=";
```

Reply:
I think in this case I prefer the spaces since its at the 'top level' of the code (if that makes sense). The other code is dealing with setting parameters for the algorithms, so `InputWorkspace='my_workspace'` which I think is good.